### PR TITLE
Fix parquet conversion tools

### PR DESCRIPTION
**Pull request by harshavardhana**
Also remove unused code from the repo, enable
staticcheck style checks for go project.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -4,13 +4,12 @@ all: check
...
 	@if [ ! -f ${GOPATH}/bin/golint ]; then echo "Installing golint" && go get -u golang.org/x/lint/golint; fi
-	@if [ ! -f ${GOPATH}/bin/gocyclo ]; then echo "Installing gocyclo" && go get -u github.com/fzipp/gocyclo; fi
+	@if [ ! -f ${GOPATH}/bin/staticcheck ]; then echo "Installing staticcheck" && go get -u honnef.co/go/tools/...; fi
 	@if [ ! -f ${GOPATH}/bin/misspell ]; then echo "Installing misspell" && go get -u github.com/client9/misspell/cmd/misspell; fi
-	@if [ ! -f ${GOPATH}/bin/ineffassign ]; then echo "Installing ineffassign" && go get -u github.com/gordonklaus/ineffassign; fi
 
 vet:
 	@echo "Running $@"
-	@go vet *.go
+	@go vet ./...
```
Comment (balamurugana) [inline Makefile:12]: This is not correct because generated go code under `gen-go` is not go format.

File: Makefile [hunk 2]
```diff
@@ -20,17 +19,14 @@ lint:
...
 	@echo "Running $@"
-	@${GOPATH}/bin/gocyclo -over 200 .
+	@${GOPATH}/bin/staticcheck *.go
 
 spelling:
 	@${GOPATH}/bin/misspell -locale US -error *.go README.md
 
-ineffassign:
-	@echo "Running $@"
-	@${GOPATH}/bin/ineffassign .
```
Comment (balamurugana) [inline Makefile:32]: Is it not useful any more?

File: columnchunk.go [hunk 3]
```diff
@@ -74,76 +74,3 @@ func pagesToColumnChunk(pages []*page) *columnChunk {
 	chunk.chunkHeader.MetaData = metaData
 	return chunk
 }
-
-func pagesToDictColumnChunk(pages []*page) *columnChunk {
```
Comment (balamurugana) [inline columnchunk.go:78]: These are all useful when nested schema supported is used.

File: Makefile (hunk 2 above)
Comment (harshavardhana) [reply Makefile:32]: No staticcheck handles this..

**Pull request closed without merging**